lobby/src/main/java/org/triplea/server/http/spark/controller/ErrorReportController.java

```diff
@@ -21,21 +21,26 @@
 
   public static final String ERROR_REPORT_PATH = "/error-report";
 
-  private final Function<ErrorReportRequest, ErrorUploadResponse> errorReportIngestion;
+  private final Function<org.triplea.server.reporting.error.ErrorReportRequest,
+      ErrorUploadResponse> errorReportIngestion;
 
   @Override
   public void run() {
     post(ERROR_REPORT_PATH, (req, res) -> uploadErrorReport(req));
   }
 
   String uploadErrorReport(final Request req) {
-    final ErrorReportRequest errorReport = readErrorReport(req);
-    final ErrorUploadResponse result = errorReportIngestion.apply(errorReport);
-    return new Gson().toJson(result);
+    final org.triplea.server.reporting.error.ErrorReportRequest errorReport = readErrorReport(req);
```

Comment:
Is it really neccessary to write the exact package here?

Reply:
This branch was created before the rename of from 'ErrorReportRequest' and 'ErrorUpload'. It's no longer necessary now.

Reply:
8b9488dd9